src/docs/sphinx/Actions/Pipelines.rst

```diff
@@ -126,7 +126,10 @@ Contour
 The contour filter evaluates a node-centered scalar field for all points at a given iso-value.
 This results in a surface if the iso-value is within the scalar field. 
 ``iso_vals`` can contain a single double or an array of doubles. 
-The code below provides examples creating a pipeline using both methods:
+Additionally, instead of specifying exact iso-values, a number of 'levels' can be entered.
+In this case, iso-values will automatically created by evenly spacing iso-values 
+through the scalar range.
```

Comment:
In this case, 'levels' number of iso-values are evenly spaced through the scalar range.